docs/book/src/reference/crd-scope.md

```diff
@@ -0,0 +1,80 @@
+# CRD Scope
+
+This section dives into the Custom Resource Definitions (CRDs), specifically focusing on their scope and how it impacts their behavior. While we'll be exploring CRD scope in detail, for managing an operator's scope (like which namespaces it monitors), refer to the dedicated operator scope documentation.
+
+## Overview
+
+CRDs come with a built-in scope field that plays a crucial role in determining the visibility and accessibility of the resulting Custom Resources (CRs). This field essentially dictates whether your CRs are cluster-wide or restricted to specific namespaces.
+
+## Reasons for Choosing Different Scopes:
+
+- **Namespace-scoped CRDs**: These are ideal when you want to limit access to CRs within certain namespaces. This is useful for scenarios like managing resources specific to a particular team or application. Additionally, you can have different versions of CRs available in different namespaces, allowing for gradual rollouts or experimentation.
+
+- **Cluster-scoped CRDs**: If you need all namespaces to have access and interact with your CRs in a uniform manner, opt for a cluster-scoped CRD. This is beneficial for shared resources or central configuration management across the entire cluster.
+
+## Setting the Scope
+
+CRD manifests are usually generated using the `operator-sdk create api` command. These manifests reside in the `config/crd/bases` directory. Within a CRD's manifest, the `spec.scope` field controls its API scope. This field accepts two valid values:
+
+- **Cluster**: This makes the CR accessible and manageable from all namespaces within the cluster.
+
+- **Namespaced**: This restricts CR access and management to the specific namespace where the CR is created.
+
+For projects employing the Operator SDK in Go, the `operator-sdk create api` command has a `--namespaced flag`. This flag determines the value of `spec.scope` and modifies the corresponding `types.go` file for the resource. In other operator types, the scope can be directly set by editing the `spec.scope` field in the CRD's YAML manifest file.
```

Comment:
We need to adapt the code since it is the Kubebuilder docs and not Operator-SDK one
Can you please ajust the format

Reply:
will this work here "The scope can be directly set by editing the `spec.scope` field in the CRD's YAML manifest file.
"

Reply:
Yes, I would recommend you:
- Follow the Quick Start: https://book.kubebuilder.io/getting-started
- Then, test out those options 

for you be able to explain and doc something you need to experiment first. The doc is just a final result of the work.

I hope that helps out.